src/Phinx/Migration/Manager.php

```diff
@@ -720,6 +739,10 @@ public function getMigrations($environment)
                     }
 
                     $versions[$version] = $migration;
+                } else {
+                    if (OutputInterface::VERBOSITY_VERBOSE <= $this->getOutput()->getVerbosity()) {
+                        $this->getOutput()->writeln("Invalid migration file " . implode(", ", $phpFiles));
```

Comment:
Should we use `<error>` for red and colorful response? Or keep plain one?

Reply:
No this is just a logging statement, let's not use `<error>` here